src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -1309,6 +1329,70 @@
 short-summary: Retrieves latest restorable timestamp for the given sql container in given region.
 """
 
+# gremlin restorable resources
+helps['cosmosdb gremlin restorable-database'] = """
+type: group
+short-summary: Manage different versions of gremlin databases that are restorable in a Azure Cosmos DB account.
```

Comment:
If the command group only contains one command, maybe you can replace the group with `cosmosdb gremlin list-restorable-database`

Reply:
Hi Yishi, we are doing this way so that we are compatible with existing Api types like SQL and MongoDB.